k8s/test/e2e/failover-demo/Dockerfile.build-distributed-object-step2

```diff
@@ -0,0 +1,23 @@
+# Copyright 2020-2022 Alibaba Group Holding Limited.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM  docker.pkg.github.com/v6d-io/v6d/vineyard-python-dev:latest
+
+WORKDIR /
+
+COPY ./build-distributed-object-step2.py /
+
+ENTRYPOINT [ "python3" ]
+
+CMD [ "build-distributed-object-step2.py" ]
```

Comment:
I'm also thinking about if those jobs can be shared between backup/failover/migration tests to avoid maintaining many similar code pieces.

Reply:
e.g., use the same dockerfile but different args or context directory when building the image.

Reply:
Done, use one basic Dockerfile to build the different docker images.